Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM debian:testing
```

Comment:
We need Ruby >= 2.2, see http://stackoverflow.com/a/40135220, and because of it we can't use Debian Jessie, see https://packages.debian.org/search?keywords=ruby.